reference/5.1/Microsoft.PowerShell.Utility/Get-Member.md

```diff
@@ -22,167 +21,232 @@ Get-Member [-InputObject <PSObject>] [[-Name] <String[]>] [-MemberType <PSMember
+GetType                   Method        type GetType()
 InitializeLifetimeService Method        System.Object InitializeLifetimeService()
-Pause                     Method        System.Void Pause()
-Refresh                   Method        System.Void Refresh()
-Start                     Method        System.Void Start(), System.Void Start(String[] args)
-Stop                      Method        System.Void Stop()
-ToString                  Method        System.String ToString()
-WaitForStatus             Method        System.Void WaitForStatus(ServiceControllerStatus desiredStatus), System.Voi...
-CanPauseAndContinue       Property      System.Boolean CanPauseAndContinue {get;}
-CanShutdown               Property      System.Boolean CanShutdown {get;}
-CanStop                   Property      System.Boolean CanStop {get;}
+Pause                     Method        void Pause()
+Refresh                   Method        void Refresh()
+Start                     Method        void Start(), void Start(string[] args)
+Stop                      Method        void Stop()
+WaitForStatus             Method        void WaitForStatus(System.ServiceProcess.ServiceControllerSt...
+BinaryPathName            Property      System.String {get;set;}
+CanPauseAndContinue       Property      bool CanPauseAndContinue {get;}
+CanShutdown               Property      bool CanShutdown {get;}
+CanStop                   Property      bool CanStop {get;}
 Container                 Property      System.ComponentModel.IContainer Container {get;}
+DelayedAutoStart          Property      System.Boolean {get;set;}
 DependentServices         Property      System.ServiceProcess.ServiceController[] DependentServices {get;}
-DisplayName               Property      System.String DisplayName {get;set;}
-MachineName               Property      System.String MachineName {get;set;}
+Description               Property      System.String {get;set;}
+DisplayName               Property      string DisplayName {get;set;}
+MachineName               Property      string MachineName {get;set;}
 ServiceHandle             Property      System.Runtime.InteropServices.SafeHandle ServiceHandle {get;}
-ServiceName               Property      System.String ServiceName {get;set;}
+ServiceName               Property      string ServiceName {get;set;}
 ServicesDependedOn        Property      System.ServiceProcess.ServiceController[] ServicesDependedOn {get;}
 ServiceType               Property      System.ServiceProcess.ServiceType ServiceType {get;}
 Site                      Property      System.ComponentModel.ISite Site {get;set;}
+StartType                 Property      System.ServiceProcess.ServiceStartMode StartType {get;}
+StartupType               Property      Microsoft.PowerShell.Commands.ServiceStartupType {get;set;}
 Status                    Property      System.ServiceProcess.ServiceControllerStatus Status {get;}
+UserName                  Property      System.String {get;set;}
```

Comment:
UserName doesn't exist in 5.1.  I don't have 6 to test if it's there either.  Probably a replicate across the articles that shouldn't be.

Reply:
Fixed